### PR TITLE
New colors supported by PS7 (#1064)

### PR DESCRIPTION
**Pull request by rjmholt**
Backport of https://github.com/PowerShell/PowerShellEditorServices/pull/1064.

### CODE REVIEW

File: src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs [hunk 1]
```diff
@@ -87,6 +87,28 @@ internal ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
                 _hostUserInterface = hostUserInterface;
             }
 
+            /// <summary>
+            /// The Accent Color for Formatting
+            /// </summary>
+            public ConsoleColor FormatAccentColor
+            {
+                get
+                { return _hostUserInterface.FormatAccentColor; }
```
Comment (SteveL-MSFT) [inline src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs]: NIT
```suggestion
                {
                    return _hostUserInterface.FormatAccentColor;
                }
```

File: src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs [hunk 2]
```diff
@@ -87,6 +87,28 @@ internal ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
...
+                set
```
Comment (SteveL-MSFT) [inline src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs]: NIT
```suggestion

                set
```

File: src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs [hunk 3]
```diff
@@ -87,6 +87,28 @@ internal ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
...
+                { _hostUserInterface.FormatAccentColor = value; }
```
Comment (TylerLeonhardt) [inline src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs]: In that case
```suggestion
                {
                    _hostUserInterface.FormatAccentColor = value;
                }
```

File: src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs (hunk 1 above)
Comment (rjmholt) [reply src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs]: This is your code @SteveL-MSFT... (see the commit)

**Pull request merged**